independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/buildtool/maven/scala/pom.tpl.qute.xml

```diff
@@ -52,7 +52,12 @@
                         <arg>-deprecation</arg>
                         <arg>-feature</arg>
                         <arg>-explaintypes</arg>
+                        {#if scala.version.startsWith("2.12.")}
```

Comment:
Is that really the exact version to use ? What if we upgrade to 2.13 ?

Reply:
We upgraded to 2.13, that's the issue. Old versions need a different path.

Reply:
Yes I get that but can't we make the template more open ended so it works with 2.13 and higher so we avoid having to keep updating ?

Reply:
@maxandersen we could make it configurable I guess and check if the variable exists, maybe that's the best backward/forward compatible fix.